test/workers/metascorer_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'sidekiq/testing'
+
+class MetascoreTest < ActionController::TestCase
+
+  setup do
+    Sidekiq::Testing.fake!
+  end
+
+  test "metascoring jobs are queued" do
+    Metascorer.jobs.clear
+    Metascorer.perform_async(11)
+    assert_equal 1, Metascorer.jobs.size
```

Comment:
Altalaban az a best practice hogy a frameworkot/library-t nem kell tesztelni, igy ez a teszt is felesleges.